### PR TITLE
Update functions.js

### PR DESCRIPTION
**Pull request by asidirop**
Disable the multilingual  library (temporarily)

### CODE REVIEW

**Comment by asidirop** [comment]
We need to keep the code.
It will be re-enabled when the site becomes multilingual again.

**Comment by asidirop** [comment]
Otherwise, we need an option for the wordpress site "is_multilingual". (This is better)

**Comment by apavlidi** [comment]
Version control systems is for that reason, git helps keep track of old changes without commending code in the codebase for historical reasons.

Multilanguage is a great feature that`s been on "issues" section here #7

File: configs/config.js [hunk 1]
```diff
@@ -49,6 +49,7 @@ const OWASP_CONFIG = {
 
 const WORDPRESS_CREDENTIALS = {
   url: 'https://www.iee.ihu.gr',
+  is_multilingual: false,
```
Comment (apavlidi) [inline configs/config.js]: The code style convection we follow is camel cased, so please try follow this for consistency.
In this case "is_multilingual" shloud be "isMultilingual"

**Comment by asidirop** [comment]
Do whatever changes on code you wish!!!!!!

**Pull request merged**